config.md

```diff
@@ -328,7 +328,7 @@ Runtime implementations MAY support any valid values for platform-specific field
 * **`linux`** (object, OPTIONAL) [Linux-specific configuration](config-linux.md).
   This MAY be set if **`platform.os`** is `linux` and MUST NOT be set otherwise.
 * **`windows`** (object, OPTIONAL) [Windows-specific configuration](config-windows.md).
-  This MAY be set if **`platform.os`** is `windows` and MUST NOT be set otherwise.
+  This MUST be set if **`platform.os`** is `windows` and MUST NOT be set otherwise.
```

Comment:
I'm not entirely clear on the maintainers' preference for this (@mrunalp [was going to document them][1]), but it's possible that they'll want `*Windows` changed to `Windows` [in the Go structure][2] now that this property is required on one platform.  At least, something like that was the reason given for not wanting pointers for [`User.UID` and `User.GID`][3].

[1]: https://github.com/opencontainers/runtime-spec/pull/759#issuecomment-301199323
[2]: https://github.com/opencontainers/runtime-spec/blob/837ee7659b0c115bb32598e4454a1f6d2460b1ea/specs-go/config.go#L29
[3]: https://github.com/opencontainers/runtime-spec/pull/759/files#diff-7f24d60f0cbb9c433e165467e3d34838L87